src/main/kotlin/no/nav/familie/ks/sak/kjerne/eøs/utenlandskperiodebeløp/UtenlandskPeriodebeløpService.kt

```diff
@@ -0,0 +1,41 @@
+package no.nav.familie.ks.sak.kjerne.eøs.utenlandskperiodebeløp
+
+import no.nav.familie.ks.sak.kjerne.eøs.felles.EøsSkjemaService
+import no.nav.familie.ks.sak.kjerne.eøs.felles.domene.EøsSkjemaRepository
+import no.nav.familie.ks.sak.kjerne.eøs.felles.endringsabonnent.EøsSkjemaEndringAbonnent
+import no.nav.familie.ks.sak.kjerne.eøs.utenlandskperiodebeløp.domene.UtenlandskPeriodebeløp
+import org.springframework.stereotype.Service
+
+@Service
+class UtenlandskPeriodebeløpService(
+    utenlandskPeriodebeløpRepository: EøsSkjemaRepository<UtenlandskPeriodebeløp>,
+    endringsabonnenter: List<EøsSkjemaEndringAbonnent<UtenlandskPeriodebeløp>>,
+) {
+    val skjemaService =
+        EøsSkjemaService(
+            utenlandskPeriodebeløpRepository,
+            endringsabonnenter,
+        )
+
+    fun hentUtenlandskePeriodebeløp(behandlingId: Long) =
+        skjemaService.hentMedBehandlingId(behandlingId)
+
+    fun oppdaterUtenlandskPeriodebeløp(
+        behandlingId: Long,
+        utenlandskPeriodebeløp: UtenlandskPeriodebeløp,
+    ) =
+        skjemaService.endreSkjemaer(behandlingId, utenlandskPeriodebeløp)
+
+    fun slettUtenlandskPeriodebeløp(
+        behandlingId: Long,
+        utenlandskPeriodebeløpId: Long,
+    ) =
+        skjemaService.slettSkjema(utenlandskPeriodebeløpId)
+
+//    @Transactional
+//    fun kopierOgErstattUtenlandskPeriodebeløp(
+//        fraBehandlingId: Long,
+//        tilBehandlingId: Long,
+//    ) =
+//        skjemaService.kopierOgErstattSkjemaer(fraBehandlingId, tilBehandlingId)
+}
```

Comment:
Kan disse kommentarene slettes, eller er det for fremtidig bruk?

Reply:
Fremtidig bruk